src/service/extensions-impl.js

```diff
@@ -440,8 +440,9 @@ export class Extensions {
     scriptElement.setAttribute('custom-element', extensionId);
     scriptElement.setAttribute('data-script', extensionId);
     const pathStr = this.win.location.pathname;
+    const base = this.win.location.protocol + '//' + this.win.location.host;
```

Comment:
`host` has the port number (`hostname` doesn't). Will send a PR addressing the two other comments soon.